main.py

```diff
@@ -178,9 +210,13 @@ def select_shoe_size(driver, shoe_size, shoe_type):
 
 def click_buy_button(driver):
     xpath = "//button[@data-qa='feed-buy-cta']"
+    class_name = "photo-component"
 
     LOGGER.info("Waiting for buy button to become clickable")
     wait_until_clickable(driver, xpath=xpath, duration=10)
+    
+    element = driver.find_element_by_class_name(class_name)
```

Comment:
Which element is this specifically?

Reply:
This is the first photo after the "Buy" button. Scrolls to photo then a little scroll up so button is visible.